shared/bash_remediation_functions/disable_prelink.sh

```diff
@@ -0,0 +1,16 @@
+function disable_prelink {
+	# Disable prelinking and don't even check
+	# whether it is installed.
+	if grep -q ^PRELINKING /etc/sysconfig/prelink
+	then
+		sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
```

Comment:
won't this replace comments as well? if I am reading this correctly if you run it twice it will replace the `# Set PRELINKING=no per...` comment iwth just `# PRELINKING=no`.

Reply:
why is this different form the rhel7 regex? shouldn't we add the comment as well?

Reply:
you might want to remove the `/g` too, this regex should replace the last occurrence and only once if I am understanding it correctly

Reply:
Good catch, I see that unshadowing requires way more brain activity than one would say at the first glace.